static/templates/settings/bot-settings.handlebars

```diff
@@ -51,6 +52,12 @@
                         </div>
                     </div>
                     <div class="input-group">
+                        <label for="create_service_base_url">{{t "Outgoing webhook service base URL" }}</label>
+                        <input type="text" name="service_base_url" id="create_service_base_url" class="required"
```

Comment:
I feel we should keep this hidden by default and attach an on change event listener to the bot-type drop down to show and hide this when needed.